packages/graphrag/graphrag/cache/factory.py

```diff
@@ -30,19 +30,22 @@ class CacheFactory(Factory[PipelineCache]):
 # --- register built-in cache implementations ---
 def create_file_cache(**kwargs) -> PipelineCache:
```

Comment:
These would need to dynamically import their storage in the creator functions too